googletest/cmake/Config.cmake.in

```diff
@@ -0,0 +1,7 @@
+@PACKAGE_INIT@
+include(CMakeFindDependencyMacro)
+set(THREADS_PREFER_PTHREAD_FLAG @THREADS_PREFER_PTHREAD_FLAG@)
+find_dependency(Threads)
```

Comment:
According to the [cmake packaging docs](https://cmake.org/cmake/help/latest/manual/cmake-packages.7.html#creating-a-package-configuration-file), since `gtest` links publicly `Threads::Threads`, then the exported `Config.cmake` file shall also call the `find_dependency(Threads)` so as to satisfy all gtest dependencies. Do you agree on this?

Reply:
Ack. Modified as suggested.